container.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+ENGINE=docker
+
+if ! which "$ENGINE" &>/dev/null ; then
+    ENGINE=podman
+fi
+
+if ! which "$ENGINE" &>/dev/null ; then
+    echo "No docker or podman in PATH" >&2
+    exit 1
+fi
```

Comment:
I think people having podman installed these days, do so because they prefer it over docker:

```suggestion
if [[ -z $ENGINE ]]; then
  if command -v podman &>/dev/null; then
    ENGINE=podman
  elif command -v docker &>/dev/null; then
    ENGINE=docker
  else
    echo "No podman or docker in PATH" >&2
    exit 1
  fi
fi
```